### ISSUE TITLE
Update peerDeps to allow React 17

### ISSUE THREAD

**Issue by JakeAdler** [reactivemarkets/react-financial-charts#468]
#### I'm submitting a...

- [x] bug
- [ ] feature
- [ ] chore

#### What is the current behavior

Unable to install without `--force` or `--legacy-peer-deps`

```console
$ npm i react-financial-charts

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: web-ui-2@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0" from react-financial-charts@1.0.1
```

#### What is the expected behavior

Should install without errors.

#### Please tell us about your environment

- Version: 1.1.0
- Browser: [all ]
- **React: 17.x**

#### Other information

From the [React 17 release blog post](https://reactjs.org/blog/2020/10/20/react-v17.html):
>We’ve only had to change fewer than twenty components out of 100,000+ in the Facebook product code to work with these changes, so we expect that most apps can upgrade to React 17 without too much trouble

I am using the library now, installed with `--legacy-peer-deps`, without any trouble. However, I'm not familiar enough with the internals of this library to know if anything might break by updating.

**Comment by markmcdowell** [comment]
I looked at this a while ago and something wasn't working so I abandoned it. I'll have another try, it may have just been storybook not supporting it.

**Comment by markmcdowell** [comment]
Not stale

**Comment by markmcdowell** [comment]
Fixed in 1.1.0
